etc/openhab2/automation/lib/hestia/defaults.js

```diff
@@ -5,14 +5,18 @@ var DEFAULTS = new java.util.HashMap();
 // C Defaults
 DEFAULTS.put("C_MIN", 0);              // Minimum C setpoint
 DEFAULTS.put("C_MAX", 40);             // Maximum C setpoint
-DEFAULTS.put("C_DEF", 22);             // Default C setpoint
+//DEFAULTS.put("C_DEF", 22);             // Default C setpoint
 DEFAULTS.put("TempC_DIFF", 0.2);       // Difference in C temps before processing the change
+DEFAULTS.put("C_COMFORT_DEF", 0.5);    // COMFORT mode hysteresis in C
+DEFAULTS.put("C_ECO_DEF", 1.0);        // Eco mode hysteresis in C
 
 // F Defaults
 DEFAULTS.put("F_MIN", 32);             // Minimum F setpoint
 DEFAULTS.put("F_MAX", 100);            // Maximum F setpoint
```

Comment:
While we're at it, can we change these to something more reasonable, like 60-80? It's a bit of a chore to tap the screen 50 times to get the temperature back when it resets.

Reply:
I used the defaults that the original code used. These are now in a config file so they can be more easily changed, but an important feature that has been added is that OH now remembers the previous state of these Items and restores them when OH restarts. The only time they should get reset is when you initially start up HestiaPi for the very first time, or when you change the TempUnit or SystemType. In all other cases, you should never have to reset them once adjusted to your preferred values. 

But I'm glad you pointed at this document because it appears to be missing some default values. The default for MinTempSetpoint for C should be 22 and for F it should be 70 and the MaxTempSetpoint for C should be 24 and for F 74. 

I'll get those values checked in right away as you will get errors when starting OH for the very first time without those defaults.